README.md

```diff
@@ -48,36 +54,43 @@ http://ryanstutorials.net/linuxtutorial/
  ```
 
 
-**2. Install RabbitMQ**
+**2. Install Erlang version 21 packages.**
 
 
  For RabbitMQ based VOLTTRON, some of the RabbitMQ specific software packages have to be installed.
 
- **a. Install Erlang packages.**
-
-  Please refer to [rabbitmq website](https://www.rabbitmq.com/which-erlang.html) to find the right version of Erlang to be installed for the version of RabbitMQ you intend to install. Also please note, RabbitMQ does not support Erlang versions older than 19.3 and newer than 20.3.x (including 21.0)
-
   **On Debian based systems:**
+  Install pre-requisites for Erlang: libwxbase, libwxgtk, and libsctpl. Search
+  and download these packages using [Ubuntu package search](https://packages
+  .ubuntu.com) and install them using dpkg -i command. For example,
+
+  ```sh
+  lsudo dbkp -i ibwxbase3.0-0v5_3.0.4+dfsg-3_amd64.deb
```

Comment:
Typo. It should be 
sudo dpkg -i libwxbase3.0-0v5_3.0.4+dfsg-3_amd64.deb

Reply:
Fixed.